assets/css/builder.css

```
@@ -304,13 +306,13 @@ div.control-table .toolbar a.builder-custom-table-button:before{line-height:17px
 .control-tabs.auxiliary-tabs>ul.nav-tabs>li,
 .control-tabs.auxiliary-tabs>div>ul.nav-tabs>li{margin-right:2px}
 .control-tabs.auxiliary-tabs>ul.nav-tabs>li>a,
-.control-tabs.auxiliary-tabs>div>ul.nav-tabs>li>a{background:white;color:#bdc3c7;border-left:1px solid #ecf0f1!important;border-right:1px solid #ecf0f1!important;border-bottom:1px solid #ecf0f1!important;padding:4px 10px;line-height:100%;-webkit-border-radius:0 0 4px 4px;-moz-border-radius:0 0 4px 4px;border-radius:0 0 4px 4px}
+.control-tabs.auxiliary-tabs>div>ul.nav-tabs>li>a{background:white;color:#bdc3c7;border-left:1px solid #ecf0f1 !important;border-right:1px solid #ecf0f1 !important;border-bottom:1px solid #ecf0f1 !important;padding:4px 10px;line-height:100%;-webkit-border-radius:0 0 4px 4px;-moz-border-radius:0 0 4px 4px;border-radius:0 0 4px 4px}
 .control-tabs.auxiliary-tabs>ul.nav-tabs>li>a>span.title>span,
 .control-tabs.auxiliary-tabs>div>ul.nav-tabs>li>a>span.title>span{margin-bottom:0;font-size:13px;height:auto}
 .control-tabs.auxiliary-tabs>ul.nav-tabs>li.active,
 .control-tabs.auxiliary-tabs>div>ul.nav-tabs>li.active{top:0}
 .control-tabs.auxiliary-tabs>ul.nav-tabs>li.active:before,
-.control-tabs.auxiliary-tabs>div>ul.nav-tabs>li.active:before{content:' ';display:block;position:absolute;width:100%;height:1px;background:#fff;top:0;left:0;top:-1px}
+.control-tabs.auxiliary-tabs>div>ul.nav-tabs>li.active:before{content:' ';display:block;position:absolute;width:100%;height:1px;background:white;top:0;left:0;top:-1px}
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

top is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)